addons/l10n_it_edi/views/l10n_it_view.xml

```diff
@@ -112,11 +112,6 @@
         <field name="model">account.move</field>
         <field name="inherit_id" ref="account.view_in_invoice_bill_tree" />
         <field name="arch" type="xml">
-            <field name="state" position="before">
-                <field name="edi_state" optional="hide"/>
-                <field name="edi_blocking_level" optional="hide"/>
-                <field name="edi_error_message" optional="hide"/>
-            </field>
```

Comment:
Unfortunately, removing this completely prevents the module from loading. 
But if you keep the `<field name="state" position="before">` and just remove what's inside, the module gets loaded correctly and the view seems fine.
Could you do this change please?

Reply:
Of course, done.